packages/fast-tooling-react/src/form/controls/control.linked-data.tsx

```diff
@@ -0,0 +1,275 @@
+
+/**
+ * Form control definition
+ */
+/* tslint:disable-next-line */
+class LinkedDataControl extends React.Component<
+    LinkedDataControlProps & ManagedClasses<LinkedDataControlClassNameContract>,
+    LinkedDataControlState
+> {
+    public static displayName: string = "LinkedDataControl";
+
+    public static defaultProps: Partial<
+        LinkedDataControlProps & ManagedClasses<LinkedDataControlClassNameContract>
+    > = {
+        managedClasses: {},
+    };
+
+    constructor(
+        props: LinkedDataControlProps & ManagedClasses<LinkedDataControlClassNameContract>
+    ) {
+        super(props);
+
+        this.state = {
+            searchTerm: "",
+            isDragging: false,
+            data: [].concat(props.value || []),
+        };
+    }
+
+    public render(): React.ReactNode {
+        // Convert to search component when #3006 has been completed
+        return (
+            <div className={this.props.managedClasses.linkedDataControl}>
+                {this.renderExistingLinkedData()}
+                {this.renderAddLinkedData()}
+            </div>
+        );
+    }
+
+    /**
+     * Render the UI for adding linked data
+     */
+    private renderAddLinkedData(): React.ReactNode {
+        return (
+            <div
+                className={
+                    this.props.managedClasses.linkedDataControl_linkedDataListControl
+                }
+            >
+                <span
+                    className={
+                        this.props.managedClasses
+                            .linkedDataControl_linkedDataListInputRegion
+                    }
+                >
+                    <input
+                        className={
+                            this.props.managedClasses
+                                .linkedDataControl_linkedDataListInput
+                        }
+                        type={"text"}
+                        aria-autocomplete={"list"}
+                        list={this.getLinkedDataInputId()}
+                        aria-controls={this.getLinkedDataInputId()}
+                        value={this.state.searchTerm}
+                        placeholder={"Add"}
+                        onChange={this.handleSearchTermUpdate}
+                        onKeyDown={this.handleLinkedDataKeydown}
+                    />
+                </span>
+                <datalist id={this.getLinkedDataInputId()} role={"listbox"}>
+                    {this.renderFilteredLinkedDataOptions()}
+                </datalist>
+            </div>
+        );
+    }
+
+    private renderFilteredLinkedDataOptions(): React.ReactNode {
+        return Object.keys(this.props.schemaDictionary).map(
+            (schemaDictionaryKey: string) => {
+                return (
+                    <option
+                        key={schemaDictionaryKey}
+                        value={this.props.schemaDictionary[schemaDictionaryKey].title}
```

Comment:
FYI using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries would mean you don't have to retrieve data from `this.props`:

```jsx
return Object.entries(this.props.schemaDictionary).map((entry: [key, value]) => {
     return    <option
                        key={entry.key}
                        value={entry.value.title}
                     />
})
```

Reply:
Nice, I like that.